src/entity-manager/LiteralEntityManager.ts

```diff
@@ -261,10 +263,11 @@ export function createLiteralEntityManager<Entity>({ connection, queryRunner }:
                 .insert()
                 .into(target)
                 .values(entity)
+                .updateEntity(options?.reload === false ? false : true)
                 .execute();
         },
 
-        update<Entity>(target: EntityTarget<Entity>, criteria: string | string[] | number | number[] | Date | Date[] | ObjectID | ObjectID[] | FindOptionsWhere<Entity>, partialEntity: QueryDeepPartialEntity<Entity>): Promise<UpdateResult> {
+        update<Entity>(target: EntityTarget<Entity>, criteria: string | string[] | number | number[] | Date | Date[] | ObjectID | ObjectID[] | FindOptionsWhere<Entity>, partialEntity: QueryDeepPartialEntity<Entity>, options?: UpdateOptions): Promise<UpdateResult> {
```

Comment:
`update` method is used to execute `UPDATE` query, it not supposed to perform complex operations, we should't add reload to this method.